hrabsence/hrabsence.php

```diff
@@ -454,7 +454,7 @@ function hrabsence_civicrm_tabs(&$tabs, $contactID) {
     )),
     'count' => $absenceDuration/(8*60),
     'title' => ts('Absences'),
-    'weight' => 300
+    'weight' => 10
```

Comment:
the default summary page in civicrm (Personal details tab) is  to 0 in the core files so I sorted the tabs before it to negative and the tabs after it to positive ... i started from -200 to give a huge space for other tabs to be before  Personal details tab  ... But I now think about it and I think ( Personal details tab ) could be added to the switch statement in hrui extension and i could give better numbering for weights of these tabs